test/test_model_locator.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import model_locator
+
+
+@pytest.mark.unit
+def test_locate_labeling_model():
+    model_path = model_locator.labeling()
+    assert model_path.endswith('.ckpt')
```

Comment:
I am not sure whether these test cases fulfill their purpose. Currently, they only assert that the environment variable has the right filename extension. I believe to test the functionality of the model_locator properly, we need to simulate different environments and assert that the correct filename is returned.

Reply:
I added a new dependency, `pytest-env` that allows to define envs for tests. 
The general problem is that other tests expect correct model paths and therefore it easily breaks other tests when both model envs are changed for tests.

I added two additional tests that cover the case where `_DEV` needs to be appended.